YoCode/ProjectBuilder.cs

```diff
@@ -73,6 +74,12 @@ private int GetReportedNumber(string keyword)
             return numbers.Count > 0 ? numbers[0] : -1;
         }
 
+        private void CleanBuildOutput(string workingDir)
+        {
+            var pr = new ProcessRunner("dotnet", workingDir, "clean");
+            pr.ExecuteTheCheck();
```

Comment:
This makes me think the name `ExecuteTheCheck` has outstayed its usefulness. It's not really a good name when we're not actually checking anything. Maybe another thing to look at in a group refactoring session?